### PR TITLE
first readme

### PR DESCRIPTION
**Pull request by nahguam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,145 @@
...
+## Start using
+
+![Maven Central](https://img.shields.io/maven-central/v/com.hotels.road/road-parent.svg)
+![GitHub license](https://img.shields.io/github/license/HotelsDotCom/data-highway.svg)
+![Build](https://travis-ci.org/HotelsDotCom/data-highway.svg?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/HotelsDotCom/data-highway/badge.svg?branch=master)](https://coveralls.io/github/HotelsDotCom/data-highway?branch=master)
+
+## Overview
+### What is Data Highway?
+The Data Highway is a service that allows data to be easily landed and consumed via HTTPS/WSS which accept and produce JSON messages. The data is first defined using a schema and a "road" is created which will accept data messages that conform to this schema. Producers of data sets thus only need to define the structure of their data and are then able to send their data to a REST endpoint and not be too concerned with what happens next. The Data Highway will ensure that this data is made available for downstream streaming consumption and also  stored reliably in a "data lake" in the cloud for access by various end users.
```
Comment (Noddy76) [inline README.md]: * `s/accept/accepts/`
* `s/The data/Data/`
* `s/data messages/messages/`
* `s/too concerned/concerned/`
* `s/The Data Highway/Data Highway/`
* `s/downstream streaming consumption/streaming consumption/`
* `s/various end users/end users/`

File: README.md [hunk 2]
```diff
@@ -1,20 +1,145 @@
...
+
+#### Architecture
+![Data Highway Architecture](data-highway-architecture.svg)
+
+##### Paver
+Paver is Data Highway's administration endpoint. It provides the following features:
+
+* Road (Synonymous with Kafka a topic) creation.
```
Comment (Noddy76) [inline README.md]: `s/Kafka a topic/Kafka topic/`

File: README.md [hunk 3]
```diff
@@ -1,20 +1,145 @@
...
+  -H "Content-Type: application/json" \
+  -d '[{"foo":"foo1","bar":"bar1"}]' \
+  https://localhost:8080/onramp/v1/roads/my_road/messages
+```
+
+### Consume messages
+
+```
+echo '{"type":"REQUEST","count":1}' |\
+  websocat -nk wss://localhost:8080/offramp/v2/roads/my_road/streams/my_stream/messages?defaultOffset=EARLIEST
```
Comment (riccardofreixo) [inline README.md]: Nice :-)

File: README.md [hunk 4]
```diff
@@ -1,20 +1,145 @@
...
+```
+
+## Examples
+Using a local instance of Test Drive, try creating road, registering a schema and producing and consuming messages using the build in user account `user:pass`.
+
+### Create a road
+
+```
+curl -sk \
+  -u user \
```
Comment (Noddy76) [inline README.md]: You'll want to mention the default password here too.

**Pull request merged**